docs/reference/hacl/kdf/index.md

```diff
@@ -33,6 +33,22 @@ The following instantiations are supported:
 
 ### API Reference
 
+**Example (BLAKE2b)**
```

Comment:
Interesting choice for an example, but it's fine. People should be able to swap it for their preferred SHA variant.

Reply:
That choice was not very well thought-out :-) I changed it to sha2-256. That's probably what most people would use.

Reply:
Changes here https://github.com/cryspen/hacl-packages/compare/49086685f2412a6dd18892124a91d8f6282188d4..0c159f06471a1370d3b2ce3796528333b6295910 (But I think they do not require another review.)